db/features.go

```diff
@@ -327,3 +327,17 @@ func (db database) GetBountiesByPhaseUuid(phaseUuid string) []Bounty {
 	db.db.Model(&Bounty{}).Where("phase_uuid = ?", phaseUuid).Find(&bounties)
 	return bounties
 }
+
+func (db database) GetTicketsByPhase(featureUuid string, phaseUuid string) ([]Tickets, error) {
```

Comment:
@MuhammadUmer44 we are to get tickets by `phaseUuid` alone not by `phaseUuid` and `featureUuid`.

Reply:
There is no concept currently where we would have phases without a feature so this logic is fine.

Reply:
@MuhammadUmer44 can you resolve this one so we can get it merged?

Reply:
@humansinstitute, Sure, I am on it.